ios/ReactNativeCameraKit/CKCameraViewComponentView.mm

```diff
@@ -165,6 +165,11 @@ - (void)updateProps:(const Props::Shared &)props oldProps:(const Props::Shared &
         _view.flashMode = [flashMode isEqualToString:@"auto"] ? CKFlashModeAuto :  [flashMode isEqualToString:@"on"] ? CKFlashModeOn : CKFlashModeOff;
         [changedProps addObject:@"flashMode"];
     }
+    id maxPhotoQualityPrioritization = CKConvertFollyDynamicToId(newProps.maxPhotoQualityPrioritization);
+    if (maxPhotoQualityPrioritization != nil && [maxPhotoQualityPrioritization isKindOfClass:NSString.class]) {
+        _view.maxPhotoQualityPrioritization = [flashMode isEqualToString:@"balanced"] ? CKMaxPhotoQualityPrioritizationBalanced :  [flashMode isEqualToString:@"quality"] ? CKMaxPhotoQualityPrioritizationQuality : CKMaxPhotoQualityPrioritizationSpeed;
+        [changedProps addObject:@"maxPhotoQualityPrioritization"];
+    }
```

Comment:
This still uses flash mode.
I would have preferred regular if-statements here since this is a little rough to read. Or at least some variables to ensure it was more readable and succinct.
Can you record a video to show that it's working/not crashing on both platforms?